api.ts

```diff
@@ -1426,9 +1458,12 @@ export type SyncFormula<
  * @param definition The definition of a formula that returns a number.
  */
 export function makeNumericFormula<ParamDefsT extends ParamDefs>(
-  definition: PackFormulaDef<ParamDefsT, number>,
+  definition: FormulaOptions<ParamDefsT, PackFormulaDef<ParamDefsT, number>>,
 ): NumericPackFormula<ParamDefsT> {
-  return Object.assign({}, definition, {resultType: Type.number as Type.number}) as NumericPackFormula<ParamDefsT>;
+  return Object.assign({}, definition, {
+    resultType: Type.number as Type.number,
+    validateParameters: wrapMetadataFunction(definition.validateParameters),
```

Comment:
If it helps, I'm definitely ok if validateParameters isn't available in the deprecated legacy makeXYZFormula() wrappers and is only offered in the builder + makeFormula. I don't think we need to continue adding new functionality to those, we can migrate 1PP code to makeFormula if that functionality is needed, and 3PPs don't use these anyway.

Reply:
Changes from this round of review did not necessitate anything extra to make the make... wrapper work and since I've gone ahead and added support already, will leave it

Reply:
I spoke too soon, it does require some, let me see what's easier

Reply:
Did you get things working with the legacy wrappers in the end?

Reply:
Yes validateParameters is still compatible with them.